experimental/segmenter/src/line.rs

```diff
@@ -113,6 +113,7 @@ pub struct LineBreakSegmenter {
 }
 
```

Comment:
Suggestion: can you move the examples in the top-level lib.rs into each of the individual structs? Then you don't need to say "Please see the module-level documentation". This would be better, and the top-level documentation can link down to each individual segmenter struct for details.

There should still be at least one top-level example, probably for either word break or line break.

Reply:
praise: nice suggestion! Will do.